modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java

```diff
@@ -1447,6 +1458,20 @@ private UserAccount _addAccountUserAccount(
 			accountId, userAccount);
 	}
 
+	private UserAccount _addUserAccount(
+			long siteId, AccountEntry accountEntry, UserAccount userAccount)
+		throws Exception {
+
+		userAccount = userAccountResource.postUserAccount(userAccount);
+
+		_userLocalService.addGroupUser(siteId, userAccount.getId());
```

Comment:
here we can call the other method (the one without the accountEntry) and leave here only the "connection". In this way we don't replicate the code